data/blog/webview2.md

```diff
@@ -0,0 +1,156 @@
+We want to provide some early information on the similarities and differences of the two solutions.
+
+---
+
+## Architecture Overview
+
+Electron and WebView2 both build from the Chromium source for rendering web content.
+Strictly speaking WebView2 builds from the Edge source, but Edge is built using a fork of the Chromium source.
+Neither Electron nor WebView2 share any DLLs files on the system with Edge or Chrome.
+
+Electron apps always bundle and distribute the exact version of Electron with which they were developed.
+WebView2 has two options in distribution.
+You can bundle the exact WebView2 library your application was developed with, or you can use a shared-runtime version that may already be present on the system.
+WebView2 provides tools for each approach, including a bootstrapping installer in case the shared runtime is missing.
+
+Applications that bundle their frameworks are responsible for updating those frameworks, including minor security releases. 
+For apps using the shared WebView2 runtime, WebView2 has its own updater, similar to Chrome or Edge, that runs independent of your application. 
+Your app will transparently inherit any updates every time it launches.
+Neither Electron nor WebView2 is managed by Windows Update.
+
+Both Electron and WebView2 inherit Chromium’s multi-process architecture - namely, a single main process that communicates with one-or-more renderer processes.
+These processes are entirely separate from other applications running on the system.
+Whether you run multiple Electron apps or multiple WebView2 apps, each application will contain a whole copy of the following process architectures:
+
+* ElectronJS Process Model: 
+    
+    ![ElectronJS Process Model Diagram](/images/Electron-Architecture.png)
+* WebView2 Based Application Process Model: 
+
+    ![WebView2 Process Model Diagram](/images/WebView2-Architecture.png)
+
+Read more about [WebView2’s process model](https://docs.microsoft.com/en-us/microsoft-edge/webview2/concepts/process-model#:~:text=WebView2%20uses%20the%20same%20process%20model%20as%20the,other%20utility%20processes%20as%20described%20in%20that%20article.) and [Electron’s process model](https://www.electronjs.org/docs/tutorial/process-model#:~:text=Process%20Model%20Electron%20inherits%20its%20multi-process%20architecture%20from,applied%20in%20the%20minimal%20quick%20start%20app%20.) here.
+
+Electron provides APIs for common desktop application needs such as menus, file system access, notifications, and more. 
+WebView2 is a component meant to be integrated into an application framework such as WinForms, WPF, or Win32. 
+WebView2 does not provide operating system APIs outside the web standard via JavaScript.
+
+Node.js is integrated into Electron.
+Electron applications may use any Node.js API, module, or node-native-addon from the render and browser processes.
```

Comment:
```suggestion
Electron applications may use any Node.js API, module, or node-native-addon from the renderer and main processes.
```

Is "node-native-addon" how Node.js refers to native modules?

Also, saying that Node.js APIs are available from the renderer is a bit misleading considering our default settings. 🤔